docs/docsite/rst/playbook_guide/playbooks_reuse_roles.rst

```diff
@@ -521,6 +523,11 @@ This syntax also runs the ``foo`` role twice;
 
 In these examples, Ansible runs ``foo`` twice because each role definition has different parameters.
 
+Using ``include_role`` or ``import_role``
+-----------------------------------------
+
+Ansible does not deduplicate imported or included roles. When you use ``include_role`` or ``import_role``, Ansible runs each role, as long as any conditions on those tasks are met. This is true even if the parameters are the same on multiple included or imported roles.
+
```

Comment:
they have a `allow_duplicates` parameter that overrides the one from the role's meta/main.yml